pkg/operator/ceph/cluster/cluster.go

```diff
@@ -131,13 +134,15 @@ func (c *cluster) doOrchestration(rookImage string, cephVersion cephver.CephVers
 	}
 
 	// Start Ceph manager
+	opcontroller.UpdateCondition(c.context, c.namespacedName, cephv1.ConditionProgressing, v1.ConditionTrue, opcontroller.ClusterProgressingReason, "Reconciling mgr(s)")
 	mgrs := mgr.New(c.context, c.ClusterInfo, *spec, rookImage)
 	err = mgrs.Start()
 	if err != nil {
 		return errors.Wrap(err, "failed to start ceph mgr")
 	}
 
 	// Start the OSDs
+	opcontroller.UpdateCondition(c.context, c.namespacedName, cephv1.ConditionProgressing, v1.ConditionTrue, opcontroller.ClusterProgressingReason, "Reconciling OSDs")
```

Comment:
```suggestion
	opcontroller.UpdateCondition(c.context, c.namespacedName, cephv1.ConditionProgressing, v1.ConditionTrue, opcontroller.ClusterProgressingReason, "Reconciling OSD(s)")
```

Reply:
All production clusters will have multiple mons and osds, so I don't think we need to make the (s) optional.